data/transactions/logic/doc.go

```diff
@@ -372,7 +371,7 @@ var txnFieldDocs = map[string]string{
 	"Sender":         "32 byte address",
 	"Fee":            "micro-Algos",
 	"FirstValid":     "round number",
-	"FirstValidTime": "Causes program to fail; reserved for future use",
+	"FirstValidTime": "Timestamp recorded on the block header at round FirstValid-1.",
```

Comment:
nit: remove the "." at the end of the line. I'm not particular whether we'll have the "." or not, but we should be consistant.

Reply:
Done. The code in opdoc aware of both dot/no dot endings.